tinymce-angular-component/src/test/ts/browser/LoadTinyTest.ts

```diff
@@ -0,0 +1,97 @@
+import '../alien/InitTestEnvironment';
+
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { Chain, Log, Pipeline, Assertions } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Arr, Strings } from '@ephox/katamari';
+import { SelectorFilter, Attr, Element, Remove } from '@ephox/sugar';
+
+import { EditorModule, EditorComponent, TINYMCE_SCRIPT_SRC } from '../../../main/ts/public_api';
+import { ScriptLoader } from '../../../main/ts/utils/ScriptLoader';
+
+UnitTest.asynctest('LoadTinyTest', (success, failure) => {
+  const cSetupEditor = (attributes: string[], providers: any[]) => Chain.async<void, void>((_, next) => {
+    @Component({
+      template: `<editor ${attributes.join(' ')}></editor>`
+    })
+    class EditorLoad { }
+
+    TestBed.configureTestingModule({
+      imports: [EditorModule, FormsModule],
+      declarations: [EditorLoad],
+      providers
+    }).compileComponents();
+
+    const fixture = TestBed.createComponent(EditorLoad);
+    fixture.detectChanges();
+
+    const editorDebugElement = fixture.debugElement.query(By.directive(EditorComponent));
+    const editorComponent = editorDebugElement.componentInstance;
+
+    editorComponent.onInit.subscribe(() => {
+      editorComponent.editor.on('SkinLoaded', () => {
+        setTimeout(() => {
+          next();
+        }, 0);
+      });
+    });
+  });
+
+  const cDeleteTinymce = Chain.op(() => {
+    ScriptLoader.reinitialize();
+
+    delete (window as any).tinymce;
```

Comment:
You could use `delete Global.tinymce` instead here, which would save having to cast window as any.